appveyor.yml

```diff
@@ -15,8 +10,6 @@ environment:
     - nodejs_version: "6"
 
 install:
-  - npm -g install npm@latest
```

Comment:
@phated If the “weird test” means using gulp-test-tool, since this tool is a thing to solve [this issue](https://www.appveyor.com/docs/lang/nodejs-iojs/#garbled-or-missing-output) on nodejs v0.10 on Windows, we can remove this tool if we skip test on Windows about old versions of nodejs.